integration_tests/build_test_containers.sh

```diff
@@ -6,7 +6,8 @@ if [[ "$(docker images -q ${D3A_IMAGE_TAG} 2> /dev/null)" == "" ]]; then
   echo "Building d3a image ..." && \
   rm -rf tests/d3a && \
   cd tests/ && \
-  git clone https://github.com/gridsingularity/d3a.git && \
+  cp -rf ~/dev/d3a . && \
+#  git clone https://github.com/gridsingularity/d3a.git && \
```

Comment:
Please revert when merging (and thanks for the tip! No need for pushing the d3a changes to github anymore :-) )

Reply:
Will do thanks!